extractor/src/main/scala/org/jetbrains/sbt/extractors/DependenciesExtractor.scala

```diff
@@ -73,17 +103,23 @@ class DependenciesExtractor(projectRef: ProjectRef,
       cs.toSeq
     }
   }
+
+  private def swapAllDerivativesOfTestConfigurations(configurations: Seq[jb.Configuration]): Set[jb.Configuration] = {
+    val jbTestConfigurations = testConfigurations.map(c => jb.Configuration(c.name))
```

Comment:
This mapping 
`testConfigurations.map(c => jb.Configuration(c.name))`
will be executed many times because the method is called in multiple places, 
But `testConfigurations` doesn't change.
We can cache the value in `lazy val` of the configuration in `org.jetbrains.sbt.extractors.DependenciesExtractor`.
Just move this 2 lines above:
`lazy val jbTestConfigurations = testConfigurations.map(c => jb.Configuration(c.name))`